Formula/mmseqs2.rb

```diff
@@ -0,0 +1,64 @@
+class Mmseqs2 < Formula
+  desc "Software suite for very fast protein sequence search and clustering"
+  homepage "https://mmseqs.org"
+  url "https://github.com/soedinglab/MMseqs2/archive/1-c7a89.tar.gz"
+  version "1-c7a89"
+  sha256 "e756a0e5cb3aa8e1e5a5b834a58ae955d9594be1806f0f32800427c55f3a45d5"
+
+  head do
+    url "https://github.com/soedinglab/MMseqs2.git"
+    resource "documentation" do
+      url "https://github.com/soedinglab/MMseqs2.wiki.git"
+    end
+  end
+
+  bottle :disable, "Requires SSE4.1 support"
+
+  option "with-avx2", "Build with AVX2 instruction set support"
+
+  needs :openmp
+  depends_on :mpi => [:cxx, :optional]
```

Comment:
Reorder needs/depends_on so we have a single block with cmake first (build dep), :arch next, :mpi then and :openmp at the end

Reply:
Reordered the section.
Brew audit now tells me to use depends_on gcc, instead of needs openmp, which I have done like suggested.